.github/workflows/build-all.yml

```diff
@@ -1,41 +1,35 @@
 name: build all
 
-env:
-  app-path: "/sps-backend" # example "/backend" for monorepos or "" for multi repos
-on:   [push]
+on: [push]
 jobs:
-  build-maven:
+  build:
     runs-on: ubuntu-latest
     env:
       TZ: Europe/Berlin # timezone
+    outputs:
+      matrix: ${{ toJSON(matrix) }}
     strategy:
+      fail-fast: false
```

Comment:
Why?

Reply:
Da war  noch das build image in einem anderen Job. Habe gedacht das es besser ist, wenn das builden überall noch weiter läuft. So sieht man gleich, ob es mehrere Fehler gibt. Ansonsten wird es gleich im Lauf abgebrochen

Reply:
Ah ok 👍 besides, anyone can change it if it bothers them